demos/deploySolution/tslib.js

```diff
@@ -0,0 +1,243 @@
+/*! *****************************************************************************
```

Comment:
I'm not clear if we need this or not?

Reply:
I haven't been able to definitively say that we don't need it, so I've left it in.

Reply:
I'm almost certain it is not needed. I don't see anything that loads it. Easy way to find out is remove it and run the demo. If it don't break, it ain't needed.

Reply:
In the PR commit, if I omit tslib.js, the app will not load.

Reply:
ok - let's leave it for now

Reply:
ok, I guess if the path is not defined, require.js just defaults to `./`. I'd say it would be better to define a path to tslib in node_modules the way you have w/ other packages.